frontend/src/v5/ui/components/legal/legalTemplate/legalTemplate.styles.tsx

```diff
@@ -0,0 +1,124 @@
+/**
```

Comment:
Can this file be a `.ts` instead of `.tsx`

Reply:
Done